### DISCUSSION TITLE
Welcome to casdoor Discussions!

### DISCUSSION THREAD

**Discussion by hsluoyz** [casdoor/casdoor#1077] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zicklag** [comment]
Hey there! 👋 I'm evaluating Casdoor for my organization and apps, and I'm really liking it's features and resource efficiency so far. Thanks for working on it!
